doc/api/webstreams.md

```diff
@@ -1457,6 +1457,48 @@ added: v16.7.0
 * Returns: {Promise} Fulfills with an `ArrayBuffer` containing the full
   contents of the stream.
 
+```mjs
+import { arrayBuffer } from 'node:stream/consumers';
+import { Readable } from 'node:stream';
+
+// Creates a 1_000_000 items array of objects
+const items = Array.from(
+  {
+    length: 1e6
```

Comment:
I think a million items per test would be too much for the whole test suite. I'd let it with only 10 items for each test so we don't compromise the overall testing performance

Reply:
I can change that to a smaller amount, I was leaving with 1M because it takes a small amount of time to load, so the user has a small delay to capture the idea